qe/plugins/org.jboss.tools.fuse.qe.reddeer/.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
```

Comment:
all projects in Fuse Tooling (and in Eclipse in general) are now 1.8

Reply:
nice catch! I need to update it in all occurrences